lib/osf-components/addon/components/resources-list/template.hbs

```diff
@@ -0,0 +1,14 @@
+<InfiniteList
```

Comment:
When we eventually have the `Add resource` button for the resource route, where would the button be invoked? Would it be invoked inside this component? Or would it be added to the route template? Asking because the button would need to have access to the `reload` action of this component. If it is invoked inside this component it would be fine. Otherwise we might need to use the manager pattern.

Reply:
I think we can just add it to this list component. I'll add that now